sdk/objc/api/peerconnection/RTCFrameEncryptorInterface.h

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2022 LiveKit
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import <Foundation/Foundation.h>
+
+#import "RTCMacros.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+RTC_OBJC_EXPORT
+@protocol RTC_OBJC_TYPE (RTCFrameEncryptorInterface)<NSObject>
+
+  // Attempts to encrypt the provided frame. You may assume the encrypted_frame
+  // will match the size returned by GetMaxCiphertextByteSize for a give frame.
+  // You may assume that the frames will arrive in order if SRTP is enabled.
+  // The ssrc will simply identify which stream the frame is travelling on. You
+  // must set bytes_written to the number of bytes you wrote in the
+  // encrypted_frame. 0 must be returned if successful all other numbers can be
+  // selected by the implementer to represent error codes.
+  // virtual int Encrypt(cricket::MediaType media_type,
+  //                     uint32_t ssrc,
+  //                     rtc::ArrayView<const uint8_t> additional_data,
```

Comment:
@cloudwebrtc Any idea what the ObjC equivalent should be for `rtc::ArrayView<const uint8_t>` ?

Reply:
I think is `NSData`.